modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/gcal/GcalKeywordReader.scala

```diff
@@ -28,11 +28,11 @@ object DummyGcalKeywordsReader extends GcalKeywordReader {
 
 object GcalKeywordsReaderImpl extends GcalKeywordReader {
 
-  def getDiffuser: SeqAction[Option[String]] = GcalEpics.instance.diffuser
+  def getDiffuser: SeqAction[Option[String]] = SeqAction(GcalEpics.instance.diffuser)
```

Comment:
I don't get why the type of `getDiffuser` didn't change if you're wrapping it on a `SeqAction`

Reply:
Because it was being implicitly converted before.

Reply:
ah ok. this is much better